app/src/main/kotlin/com/twain/interprep/presentation/ui/components/note/NoteCard.kt

```diff
@@ -65,44 +65,42 @@ fun NoteCard(
                     interview = interview
                 )
             }
-            if (notes.isNotEmpty()) {
+            notes.firstOrNull()?.let { note ->
                 Divider()
-                notes.forEachIndexed { index, note ->
-                    Row(
+                Row(
+                    modifier = Modifier
+                        .padding(top = dimensionResource(id = R.dimen.dimension_16dp))
+                        .clickable { onDeleteNoteClick(note) }
+                ) {
+                    Box(
                         modifier = Modifier
-                            .padding(top = dimensionResource(id = R.dimen.dimension_16dp))
-                            .clickable { onDeleteNoteClick(note) }
+                            .size(dimensionResource(id = R.dimen.dimension_32dp))
+                            .clip(CircleShape)
+                            .background(Purple100),
+                        contentAlignment = Alignment.Center
                     ) {
-                        Box(
-                            modifier = Modifier
-                                .size(dimensionResource(id = R.dimen.dimension_32dp))
-                                .clip(CircleShape)
-                                .background(Purple100),
-                            contentAlignment = Alignment.Center
-                        ) {
-                            Text(
-                                text = index.toString(),
-                                style = MaterialTheme.typography.titleMedium
-                            )
-                        }
-                        Spacer(modifier = Modifier.width(dimensionResource(id = R.dimen.dimension_8dp)))
-                        Column(
-                            verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.dimension_4dp))
-                        ) {
-                            Text(
-                                text = note.interviewSegment,
-                                style = MaterialTheme.typography.bodyLarge
-                            )
+                        Text(
+                            text = 0.toString(),
+                            style = MaterialTheme.typography.titleMedium
```

Comment:
Do we need to keep the index in the front of each note? If we are not keeping it, what is the ui to show?

<img src="https://github.com/Audienix/InterPrep/assets/67870437/02e64f45-cd3e-44fe-8395-5aac615b8d78" width=200>

Reply:
We will keep the index.

Reply:
btw I think the index should start from 1?

Reply:
I was about to write the same thing. Index should start from 1